### PR TITLE
support kafka

### PR DESCRIPTION
**Pull request by philsong**

### CODE REVIEW

File: setup-env.sh [hunk 1]
```diff
@@ -0,0 +1,3 @@
+export PYTHONPATH="$PWD"
```
Comment (gavincyi) [inline setup-env.sh]: Why needed?

File: subscription.ini [hunk 2]
```diff
@@ -0,0 +1,18 @@
+[Bitfinex-BTCUSD-WS]
```
Comment (gavincyi) [inline subscription.ini]: It looks to me the change in subscription.ini is irrelevant.

**Comment by philsong** [comment]
support kkex, aex, bibox, kafka

File: setup-env.sh (hunk 1 above)
Comment (philsong) [reply setup-env.sh]: source setup-env.sh
python ./befh/bitcoinexchangefh.py -kafka -kafkadest "127.0.0.1:9092" -instmts subscription.ini -output log.txt

**Pull request closed without merging**